airbyte-integrations/connectors/source-stripe/unit_tests/test_streams.py

```diff
@@ -170,40 +170,181 @@ def test_lazy_substream_data_is_filtered(
+                    "created": 1679568588,
+                    "status": "available",
+                },
+            ],
+            [{"created[gte]": 1631199615, "created[lte]": 1662735615}, {"created[gte]": 1662735616, "created[lte]": 1692802815}],
+            "balance_transactions",
+            "full_refresh",
+            {},
+        ),
+        (
+            {
+                "/v1/balance_transactions": [
+                    {
+                        "json": {
+                            "data": [balance_transactions_api_objects[-1]],
+                            "has_more": False,
+                        }
+                    },
+                ],
             },
+            [
+                {
+                    "id": "txn_tiJtvvhF7ox3YEmKvVQhfEcX",
+                    "object": "balance_transaction",
+                    "amount": -9164,
+                    "created": 1679568588,
+                    "status": "available",
+                },
+            ],
+            [{"created[gte]": 1665308989, "created[lte]": 1692802815}],
+            "balance_transactions",
+            "incremental",
+            {"created": 1666518588},
+        ),
+        (
             {
-                "json": {
-                    "data": [
-                        {"id": "txn_tiJtvvhF7ox3YEmKvVQhfEcX", "object": "balance_transaction", "amount": -9164, "status": "available"}
-                    ],
-                    "has_more": False,
-                }
+                "/v1/refunds": [
+                    {
+                        "json": {
+                            "data": [refunds_api_objects[0]],
+                            "has_more": False,
+                        }
+                    },
+                    {
+                        "json": {
+                            "data": [refunds_api_objects[-1]],
+                            "has_more": False,
+                        }
+                    },
+                ],
             },
-        ],
-    )
+            [
+                {
+                    "id": "re_3NYB8LAHLf1oYfwN3EZRDIfF",
+                    "object": "refund",
+                    "amount": 100,
+                    "charge": "ch_3NYB8LAHLf1oYfwN3P6BxdKj",
+                    "created": 1653299388,
+                    "currency": "usd",
+                },
+                {
+                    "id": "re_Lf1oYfwN3EZRDIfF3NYB8LAH",
+                    "object": "refund",
+                    "amount": 15,
+                    "charge": "ch_YfwN3P6BxdKj3NYB8LAHLf1o",
+                    "created": 1679568588,
+                    "currency": "eur",
+                },
+            ],
+            [{"created[gte]": 1631199615, "created[lte]": 1662735615}, {"created[gte]": 1662735616, "created[lte]": 1692802815}],
+            "refunds",
+            "full_refresh",
+            {},
+        ),
+        (
+            {
+                "/v1/refunds": [
```

Comment:
I would have expected `created[gte]` and `created[lte]` query params here. Is there a reason it's not there?

Reply:
Well they should be but that's not so explicit and the absence of URL params would not fail the test. So I added an explicit check